perspective.el

```diff
@@ -1259,7 +1260,8 @@ PERSP-SET-IDO-BUFFERS)."
              (append
               (list
                (format "Switch to buffer (%s): " (persp-current-name))
-               (cl-remove-if #'null (mapcar #'buffer-name (persp-current-buffers)))
+               (cl-remove-if #'null (mapcar #'buffer-name (-intersection (buffer-list)
```

Comment:
I'm really confused how `-intersection` helps sort by access time here. Please clarify.

Reply:
By default (buffer-list) is sorted by access time and the order is kept in the -intersection operation.

Reply:
Very clever! Please add a code comment with this explanation.